### PR TITLE
Created dec 2018 monthly update

### PR DESCRIPTION
**Pull request by rubymorillo**
Blog post roundup for Dec 2018

### CODE REVIEW

File: _posts/2019-02-02-december-2018-updates.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+author: Stephanie Morillo
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in October.
+
+## rubygems.org news
+
+In RubyGems.org news, lead RubyGems.org maintainer [@dwradcliffe](https://github.com/dwradcliffe) completed porting the production RubyGems.org deployment to use Kubernetes! This didn’t cause any user-facing changes, but makes it easier for others to develop locally, and will help us with our efforts to avoid downtime.
+
+Another major development was enabling two factor authentication, a Google Summer of Code project that can greatly improve the security of logging in and publishing new gems, for any gem authors who enable it.
```
Comment (sonalkr132) [inline _posts/2019-02-02-december-2018-updates.md]: Can we please ecnelises add here? Link to guide would be nice too 
https://guides.rubygems.org/setting-up-multifactor-authentication/

Comment (rubymorillo) [reply _posts/2019-02-02-december-2018-updates.md]: @sonalkr132 I added a link to the guide, as requested. For my FYI, what is "ecnelises"?

Comment (sonalkr132) [reply _posts/2019-02-02-december-2018-updates.md]: https://github.com/ecnelises is the gsoc student who worked on this. We did credit him in Nov's update under rubygems. I guess twice is better than once :sweat_smile:

Comment (rubymorillo) [reply _posts/2019-02-02-december-2018-updates.md]: @sonalkr132 added; thanks Aditya!

**Comment by indirect** [comment]
Thanks, @rubymorillo and @sonalkr132!

**Pull request merged**